README.rst

```diff
@@ -39,6 +39,12 @@ Add the tables to the db::
 
 **Warning**: An initial data fixture is included that creates an admin_sso.Assignment to assign any user with an incuna.com email to the Admin user.
 
+To allow anonymous access to urls, like external link rewrites, add ``LOGIN_EXEMPT_URLS`` in ``settings.py``::
```

Comment:
I'd like to mention a common thing that you would want to allow anonymous access to, which is inaccessible by default when using `LoginRequiredMiddleware `.